docs_src/microservices/device/profile/Ch-DeviceProfile.md

```diff
@@ -1,10 +1,12 @@
 # Device Profile
 
-The device profile describes a type of device within the EdgeX system. Each
+The device profile describes a type of [device](../../../general/Definitions.md#device) within the EdgeX system. Each
 device managed by a device service has an association with a device profile,
 which defines that device type in terms of the operations which it supports.
 
-For a full list of device profile fields and their required values see the [Device Profile Reference](./Ch-DeviceProfileRef.md)
+For a full list of device profile fields and their required values see the [device profile reference](./Ch-DeviceProfileRef.md).
+
+For a detailed look at the device profile model and all its properties, see the [metadata device profile data model](../../core/metadata/Ch-Metadata.md#data-models).
```

Comment:
@cloudxxx8 - all the pages have been updated with new models.  You are only seeing the old model because the PR for that pages updates hasn't been merged yet.  We are preserving all the links so that the pages all work the same, but you have to pull down the update PR #193